app/services/metrics/content_distribution_metrics.rb

```diff
@@ -3,6 +3,7 @@ class ContentDistributionMetrics
     def count_content_per_level
       counts_by_level.each_with_index do |count, level|
```

Comment:
could do the `to_enum.with_index(1) ` like in the test in the previous commit

Reply:
I probably should have done this in a separate commit, but I did make this change in 6cb6857b4a0bf29488e96919e4b6ff2b4b01487d